interface/src/config.js

```diff
@@ -0,0 +1,112 @@
+export const MIC_SAMPLE_RATE = 16000;
+export const BLOCK_SIZE = 8192;
+export const MODEL_OPTIONS = [
+  "tiny",
+  "base",
+  "small",
+  "medium",
+  "large",
+  "large-v1",
+  "large-v2",
+];
+export const SUPPORTED_LANGUAGES = [
```

Comment:
This list is not the same as the backend's one https://github.com/saharmor/whisper-playground/pull/23/files#diff-651976572b51d1a16ede6a1c4c1be9a1482605ce57ca58fbfca25e4a55c43210R15